github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -23,30 +23,6 @@ periodics:
       - /usr/bin/flakefinder
       name: ""
       resources: {}
-- annotations:
-    testgrid-create-test-group: "false"
-  cluster: kubevirt-prow-control-plane
-  cron: 45 0 * * *
-  decorate: true
-  labels:
-    preset-gcs-credentials: "true"
-    preset-github-credentials: "true"
-  name: periodic-publish-kubevirt-flakefinder-weekly-report
```

Comment:
I find the weekly report useful to get a good picture of some of the more prevalent flakes.